src/database/mixxxdb.cpp

```diff
@@ -30,6 +33,45 @@ const QString kUserName = QStringLiteral("mixxx");
 
 const QString kPassword = QStringLiteral("mixxx");
 
+bool ensureColumnExists(const QSqlDatabase& database,
+        const QString& tableName,
+        const QString& columnName,
+        const QString& alterSql) {
+    QSqlQuery pragmaQuery(database);
+    pragmaQuery.prepare(QStringLiteral("PRAGMA table_info(%1)").arg(tableName));
+    if (!pragmaQuery.exec()) {
+        kLogger.warning() << "Failed to inspect table" << tableName
+                          << pragmaQuery.lastError().text();
+        return false;
+    }
+
+    while (pragmaQuery.next()) {
+        if (pragmaQuery.value(QStringLiteral("name")).toString() == columnName) {
+            return true;
+        }
+    }
+
+    QSqlQuery alterQuery(database);
+    if (!alterQuery.exec(alterSql)) {
+        kLogger.warning() << "Failed to add column" << columnName << "to" << tableName
+                          << alterQuery.lastError().text();
+        return false;
+    }
+
+    kLogger.info() << "Added missing column" << columnName << "to table" << tableName;
+    return true;
+}
+
+bool ensureTuningFrequencyColumn(const QSqlDatabase& database) {
```

Comment:
This is redundant. We control that columns exist via `version="40"` 
It does not really make sens to just create the missing column here if we don't do it for other columns.